rjc_account_form/reports/invoice_layout.xml

```diff
@@ -0,0 +1,127 @@
+<odoo>
+
+    <template id="report_assets_common" name="Invoice common assets" inherit_id="web.report_assets_common">
+        <xpath expr="." position="inside">
+            <link rel="stylesheet" type="text/scss" href="/rjc_account_form/static/scss/style_invoice_report.scss"/>
+        </xpath>
+    </template>
+
+    <template id="minimal_layout" inherit_id="web.minimal_layout">
+        <xpath expr="//t[@t-if='subst_needed']" position="replace">
```

Comment:
move on javascript to folder 

> /static/src/js/invoice_preprint.js

and call javascript like a stylesheet (scss)